sim/core/stats/stats.go

```diff
@@ -332,6 +332,9 @@ type PseudoStats struct {
 	DamageDealtMultiplier       float64            // All damage
 	SchoolDamageDealtMultiplier [SchoolLen]float64 // For specific spell schools (arcane, fire, shadow, etc).
 
+	// Apply Aura: Mod Melee Critical Damage % (Arcane, Fire, Frost, Holy, Nature, Physical, Shadow)
+	SchoolCritMultiplier [SchoolLen]float64 // For specific spell schools (arcane, fire, shadow, etc).
```

Comment:
I don't see this used anywhere

Reply:
Hodir Stormpower buff the comment is how the wow db has it listed in the aura effects

Reply:
Just noticed The hodir_ai file is not shown by default from github due to its size so probably missed it